matrix_sdk/src/client.rs

```diff
@@ -946,6 +950,40 @@ impl Client {
         self.store().get_room(room_id).and_then(|room| room::Left::new(self.clone(), room))
     }
 
+    /// Tries to decrypt a `AnyRoomEvent`. Returns unencrypted room event when
+    ///  decryption fails.
+    #[cfg(feature = "encryption")]
+    #[cfg_attr(feature = "docs", doc(cfg(encryption)))]
+    pub async fn decrypt_room_event(&self, event: &AnyRoomEvent) -> RoomEvent {
```

Comment:
This method should indeed return the `MegolmError` if it's going to be public, otherwise people can't inspect why decryption failed. If you don't want to deal with this currently, feel free to just remove the `pub`.